include/aspect/material_model/steinberger.h

```diff
@@ -52,6 +52,10 @@ namespace aspect
     class Steinberger: public MaterialModel::Interface<dim>, public ::aspect::SimulatorAccess<dim>
     {
       public:
+        /**
+         * Virtual destructor.
+         */
+        virtual ~Steinberger();
```

Comment:
My Eclipse (Neon 4.6.0) creates a warning for the whole class (and all other material models), because it believes the destructor is not virtual. May well be a bug inside eclipse, I mostly introduced this to get rid of the warning. I guess it is better to keep the material models uniform, so I will remove the line.